cosas_helm

```diff
@@ -0,0 +1,29 @@
+  helm install --name database stable/postgresql
```

Comment:
@avara1986 whas this voluntary?

Reply:
No, this files are my notes to install a postgrsql in the cluster, i've removed it